File: tests/fixtures/corpus/src/java/Cases.java
class Cases {
  void structured() {
    throw new ApiException("JAVA_USER_MISSING", "User was not found");
  }

  void dynamic(String dynamicCode) {
    throw new ApiException(dynamicCode, "Dynamic code remains unresolved");
  }

  void generic() {
    throw new IllegalStateException("Invalid state");
  }

  Exception noise() {
    return new IllegalArgumentException("JAVA_NOISE");
  }
}

File: tests/fixtures/corpus/src/csharp/Cases.cs
class Cases
{
    void Structured()
    {
        throw new ApiException("CS_USER_MISSING", "User was not found", 404);
    }

    void Dynamic(string dynamicCode)
    {
        throw new ApiException(dynamicCode, "Dynamic code remains unresolved", 500);
    }

    void Generic()
    {
        throw new InvalidOperationException("Invalid state");
    }

    Exception Noise() => new InvalidOperationException("CS_NOISE");
}
